docs/reST/ref/math.rst

```diff
@@ -195,6 +195,36 @@ Multiple coordinates can be set using slices or swizzling
 
       .. ## Vector2.distance_squared_to ##
 
+   .. method:: move_towards
+
+      | :sl:`returns a vector that moves toward to the given vector at the given speed.`
+      | :sg:`move_towards(Vector2, float) -> Vector2`
+
+      Similar to ``lerp()``, this returns a Vector which moves self to the
+      given Vector at a given speed. The first parameter determines the target
```

Comment:
Still lots of mentions of speed that should be distance.

Reply:
Sorry, it looks like my editor didn't catch that.